types/svelte-leafletjs/package.json

```diff
@@ -1,6 +1,7 @@
 {
     "private": true,
     "dependencies": {
+        "@types/react-leaflet": "^2.8.3",
```

Comment:
`@types/svelte-leafetjs` only depends on `@types/react-leaflet` to reduce the maintenance burden of maintaining the same [enormous list of Events](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/64bd0fe50789b2e6372b6713e8ddcd03d62896b7/types/react-leaflet/index.d.ts#L18-L109) in both packages. It's much better (imo) to have a single source of truth. [This convoluted syntax](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/svelte-leafletjs/index.d.ts#L33-L37) parses the react definitions and converts them into an interface for the svelte definitions. 

So, in reality there is no actual dependency between these two packages. 


If `react-leaflet` is being deleted, I think it would make sense to copy [all this code](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/64bd0fe50789b2e6372b6713e8ddcd03d62896b7/types/react-leaflet/index.d.ts#L18-L109) into the svelte package, rather than making the svelte package depend on a deprecated npm module which can't be updated.

Thoughts?

Reply:
@MysteryBlokHed

Reply:
That'd probably be a good idea since the `@types/react-leaflet` package will no longer be changeable after deletion.